### ISSUE TITLE
Build fails with Gradle 2.5

### ISSUE THREAD

**Issue by WonderCsabo** [xujiaao/AARLinkSources#8]
If a project is built with Gradle 2.5, the build will fail with the following exception:
```
Could not find method executeWithoutThrowingTaskFailure() for arguments [] on task ':aarLinkSources'
```

This is due to a change in Gradle 2.5 : https://github.com/gradle/gradle/commit/f58d618aa912955f01bee42bba828a3b9c3924b3

I already have an easy fix:
```groovy
try {
	aarLinkSourcesTask.execute()
} catch (e) {
	// maybe log?
}
```

However i think it would be even better to only execute this plugin if it is in tooling mode, eg. when called by the IDE. This is totally unnecessary when called from command line. I am waiting for [feedback](https://discuss.gradle.org/t/plugin-is-in-tooling-mode/10503) on that.

**Comment by hzsweers** [comment]
@WonderCsabo your fix isn't really a fix though is it? It just catches the exception and proceeds without actually achieving the purpose of the task?

**Comment by pcqpcq** [comment]
i don't think this fix is a good idea

**Comment by WonderCsabo** [comment]
The code above would replace [this](https://github.com/xujiaao/AARLinkSources/blob/master/plugin/src/main/groovy/com/xujiaao/android/idea/AARLinkSourcesPlugin.groovy#L49) line.

@hzsweers this is actually a fix, because it calls the `execute()` method, which is available in Gradle 2.5 instead of `executeWithoutThrowingTaskFailure()`, which is not.

@pcqpcq well, the code above is ugly, but actually results in the same behavior as the old method. The old method also catched all exceptions silently, and with a reason: an exception in this plugin should not break the build.

**Comment by hzsweers** [comment]
This library is no longer necessary, as sources attachment is now included in android studio as of 1.4 beta 3.

https://code.google.com/p/android/issues/detail?id=59220
